libs/watt/package/textarea-field/watt-textarea-field.component.ts

```diff
@@ -41,35 +42,39 @@ import { WattFieldComponent } from '@energinet/watt/field';
   selector: 'watt-textarea-field',
   styleUrls: ['./watt-textarea-field.component.scss'],
   encapsulation: ViewEncapsulation.None,
-  template: `<watt-field [label]="label" [control]="formControl">
+  template: `<watt-field [label]="label()" [control]="formControl()">
     <textarea
-      [attr.placeholder]="placeholder"
-      [value]="value"
-      [(ngModel)]="model"
-      [disabled]="isDisabled"
-      (ngModelChange)="onChange($event)"
-      [required]="required"
+      [attr.placeholder]="placeholder()"
+      [value]="value()"
+      [disabled]="isDisabled()"
+      [required]="required()"
+      (input)="onInput($event)"
     ></textarea>
     <ng-content ngProjectAs="watt-field-hint" select="watt-field-hint" />
     <ng-content ngProjectAs="watt-field-error" select="watt-field-error" />
   </watt-field>`,
+  host: {
+    '[attr.watt-field-disabled]': 'isDisabled()',
+  },
 })
 export class WattTextAreaFieldComponent implements ControlValueAccessor {
-  @Input() formControl!: FormControl;
-  @Input() value!: string;
-  @Input() placeholder?: string;
-  @Input() required = false;
-  @Input() label!: string;
-
   /** @ignore */
-  model!: string;
+  private element = inject(ElementRef);
+  formControl = input.required<FormControl>();
+  placeholder = input<string>();
+  required = input(false);
+  label = input<string>();
+
+  onInput(event: Event) {
+    const target = event.target as HTMLTextAreaElement;
+    this.value.set(target.value);
+  }
 
   /** @ignore */
-  private element = inject(ElementRef);
+  value = model<string>('');
 
   /** @ignore */
-  @HostBinding('attr.watt-field-disabled')
-  isDisabled = false;
+  isDisabled = signal(false);
 
   /** @ignore */
   onChange: (value: string) => void = () => {
```

Comment:
Remove this, since it is no longer needed

Reply:
Removed